frontend/javascripts/oxalis/view/action-bar/download_modal_view.tsx

```diff
@@ -507,7 +506,7 @@ function _DownloadModalView({
               >
                 {!hasVolumes ? "This is a Skeleton-only annotation. " : ""}
                 {!hasSkeleton ? "This is a Volume-only annotation. " : ""}
-                {messages["annotation.download"]}
+                {messages["download.data_is_ready_for_download"]({ typeName })}
```

Comment:
This tab is only ever shown for annotations, so the distinction is not needed here. The resulting text "The following **dataset data** is available for download immediately." would sound weird, too, so I'd revert that change.

Reply:
Thanks for :cat:-ching that. I revered the change :recycle: